README.md

```diff
@@ -1,10 +1,25 @@
 # StudentManagement
-Student Information Management tool
-We have provided you with a student growth recording tool that can record your grades, volunteer experiences, and related documents. In order to use it, you must complete the following configuration
+Student Information Management Platform
+
+This is a student information management platform designed for course lecturer and academic affairs office. The major functionality includes course grades recording, volunteering experiences recording, and documentations management.
+
+## Installation Instructions
+
+`TODO: rewrite the following instructions with specific details, for example, which version of Windows, which specific version of JDK needed, what are the [relevant libraries] and [encoding, decoding] tools, how to compile and run, etc.`
 
 1. The software is currently only tested on Windows systems and may not be compatible with other systems
 2. The computer is equipped with JDK
 3. Install relevant libraries and encoding and decoding tools
 4. You can compile and run on the terminal or import projects in the IDE
 
+### Dependance Installation
```

Comment:
Could you fix the typo `Dependance`

Reply:
https://github.com/XavierBohuaLiu/StudentManagement/pull/1/commits/1daf564ca61b2ee8563f31cf154397e3ce892be7